app/src/main/java/com/example/fooding/activities/LoginActivity.java

```diff
@@ -29,10 +30,10 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_login);
         currentUser = ParseUser.getCurrentUser();
-
-//        if(currentUser != null){
-//            goMainActivity();
-//        }
+//
+        if(currentUser != null){
+            goMainActivity();
```

Comment:
renaming still left open

Reply:
I updated it from my laptop, then pushed to GitHub